### ISSUE TITLE
Inner/Outer glow not working

### ISSUE THREAD

**Issue by moooofddg** [photopea/photopea#6646]
Your Operating System: windows
Your Web Browser: google
The problem happens in the Incognito Mode, too:

**This must be done to see the problem**
1. Go to www.photopea.com
2. Open project
3. Place on image or use brush to draw
4. Go to Blending options
5. Tick inner/outer glow box 
6. Blue box is ticked but nothing happens the picture/paint
7. Adjust glow settings maybe that might work
8. still nothings happens

**Comment by Epikoofy** [comment]
same this has been happening to me it's very frustrating

**Comment by photopea** [comment]
sorry, it should be fixed now.

**Comment by WarisMaqbool-ship-it** [comment]
@photopea sorry if this might come across as out of topic or maybe it is not the right place to ask this but I was trying to implement the spread slider in my own implementation of inner glow, I did an experiment with Photopea's inner glow where I created a 100x100 solid black rectangle applied inner glow layer style on it with varying parameters:

At size 100px, and spread 0% with range 100%(I assume it refers to the percentage of the effect being affected by the contour curve), Gradient: pure white rgb(255,255,255), opacity 100%,  jitter: 0%, the gradient of the very first pixel from the edge is rgb (127,127,127) which diffuses to rgb(0,0,0) towards the shape center. Assuming all the other parameter are same and we just change the spread, at spread: 25%, the gradient of the very first pixel changes to rgb (169,169,169) and diffuses down to rgb(0,0,0) at the shape center, and at spread 50% the gradient values goes from rgb(254,254,254) all the way to rgb(1,1,1) at the very center, this point's at exponential growth of the gradient at the edges respective to spread slider, but the gradient in turn diffuses almost close to rgb (0,0,0) towards the shape center. What happens after 50 is interesting though, for example at spread 55%, pixels till certain distances from the edge are receiving a solid white gradient (rgb (255,255,255)) and then diffusing till rgb(1,1,1) towards the center. What I mean to say is after spread reaches greater than 50%, the pure solid boundary of the effect seems to be increasing. Before spread is greater than 50%, there is no solid boundary in the whole effect. So my question really is, is it just that the spread slider is supposed to behave differently before and after value 50? Or is it some elegant mathematical equation leading to both of these behaviours? I know this software is closed source but it is something that has been bothering me and would appreciate if I get some clarification regarding it.
